articles/fleetctl.md

```diff
@@ -10,25 +10,18 @@ fleetctl also provides a quick way to work with all the data exposed by Fleet wi
 
 ## Installing fleetctl
 
-Download and install [Node.js](https://nodejs.org/en).
 
-Install fleetctl with npm (included in Node.js).
+To install fleetctl on macOS, Windows, or Linux, download the fleectl binary from [GitHub](https://github.com/fleetdm/fleet/releases). 
 
-```sh
-sudo npm install -g fleetctl
-```
-
-To install fleetctl on Windows or Linux, download the fleectl binary from [GitHub](https://github.com/fleetdm/fleet/releases).
+After downloading, extract the binary. To run fleetctl commands, use the binary's path (`/path/to/fleetctl`). For convenience, copy the binary to a directory in your `$PATH` (ex: `/usr/local/bin`). This allows you to execute fleetctl from anywhere without specifying its location.
```

Comment:
What are the steps/commands the user needs to run to extract and coy to path? Reminder that we want to make this as easy as possible for someone totally new to IT.

Reply:
Added some minor wording updates. I don't think any more detail should be needed. Extracting the contents of zip files and copying/moving files are fairly basic and common actions, not something specific to IT (or new to IT) professionals.

Reply:
I don't know how to copy the binary to a directory in my `$PATH` 🙈  I would guess I'm not the only one at Fleet.

Also, to upgrade, it is just install the new binary? Or, do I have to install the binary and update my path?

Thanks for bearing with me @allenhouchins. I think we want to make things dead simple. Happy to spend some of our unpacking the why time tomorrow to get this one merged in.

Reply:
@allenhouchins just giving you another ping! here.

> I don't know how to copy the binary to a directory in my $PATH 🙈 I would guess I'm not the only one at Fleet.
> 
> Also, to upgrade, it is just install the new binary? Or, do I have to install the binary and update my path?

Please let me know if I can be helpful addressing these.

Reply:
FYI @lukeheath

Reply:
Added changes. Please review!